android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/INIParser.java

```diff
@@ -0,0 +1,123 @@
+package com.tavultesoft.kmea.util;
+
+import java.util.Map;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * A custom, lightweight INI parser designed initially for use with Keyman's .kmp and .inf files.
+ * @author jahorton
+ *
+ */
+public class INIParser {
```

Comment:
Fun detail - Android Studio requires any desired test classes and tests to be declared `public`.  This differs from the situations I've experienced in other IDEs.

If a test class itself is not public, Android Studio won't realize that it exists as a testing class.  Caused me quite the headache for a while.